gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/ITGcsNio.java

```diff
@@ -0,0 +1,351 @@
+package com.google.gcloud.storage.contrib.nio;
```

Comment:
In gcloud-java modules, we moved integration tests to a separate package (e.g. com.google.gcloud.storage.contrib.nio.<b>it</b> in this case) so that we could test access issues (e.g. accidentally keeping a public method package-scope).  Do you think that'd be valuable here?